pkg/daemon/ceph/osd/agent.go

```diff
@@ -615,8 +623,9 @@ func isBluestoreDevice(cfg *osdConfig) bool {
 }
 
 func isBluestoreDir(cfg *osdConfig) bool {
-	// A dir will use filestore unless explicitly requested to be bluestore
-	return cfg.dir && cfg.storeConfig.StoreType == config.Bluestore
+	// All directory OSDs are filestore
+	// TODO: is this change safe?
```

Comment:
Should we go ahead and remove the code that expects true from this call? Or you want to save it for another day?

Reply:
I'd rather save it for when the whole new-filestore-disk-osd paths are ripped out